src/include/cell.i

```diff
@@ -943,6 +1047,8 @@ __wt_cell_unpack_dsk(
             F_SET(unpack, WT_CELL_UNPACK_TIME_PAIRS_CLEARED);
             if (unpack->stop_ts == WT_TS_MAX)
                 unpack->stop_ts = WT_TS_NONE;
+            if (unpack->durable_stop_ts == WT_TS_MAX)
```

Comment:
This scenario is not possible now because of the default value of `WT_TS_NONE` for durable timestamps. This code can be removed.

Reply:
Removed.